### ISSUE TITLE
Dataset information

### ISSUE THREAD

**Issue by marcomameli1992** [Jindi0/SAFE#5]
From the textual dataset which of the json files do you use to obtain text?
I think that the news is inside the news_article file, it the right file?

Thank you for your answer

**Comment by Jindi0** [comment]
Yes, please check the "text" and "title" keys in the news_article file.
